### ISSUE TITLE
:face_vomiting: emoji adds "," char

### ISSUE THREAD

**Issue by DavidLibeau** [ludumdare/ludumdare#1455]
Writing ":face_vomiting:" emoji (seen on game description) automatically adds the char ",". So that does not interpret the emoji but output ":face,_vomiting:".

**Comment by zwrawr** [comment]
Didn't i just fix that

**Comment by povrazor** [comment]
I .. thought we did?

**Comment by zwrawr** [comment]
Maybe i broke it

**Comment by DavidLibeau** [comment]
Ho, I saw the bug yesterday, so maybe it's ok now.

**Comment by zwrawr** [comment]
Okay, #1438 probably fixed it.
I think :face_vomiting: isn't an emoji btw.

**Comment by zwrawr** [comment]
So it was added to Unicode 10.0, we should update our emoji's

https://www.emojione.com/blog/introducing-unicode-100

**Comment by DavidLibeau** [comment]
The emoji is not interpreted here : https://ldjam.com/events/ludum-dare/40/motion-only
:face_vomiting: is an emoji listed here https://emoji.codes/#

**Comment by povrazor** [comment]
Yeah, I haven't updated the names database yet, so we're missing some of the most recent.

**Comment by DavidLibeau** [comment]
Ok !

**Comment by povrazor** [comment]
Re-branding this issue. May as well have one for this task too.
